src/librustc_typeck/diagnostics.rs

```diff
@@ -934,6 +934,28 @@ The number of supplied parameters much exactly match the number of defined type
 parameters.
 "##,
 
+E0088: r##"
+You gave too many lifetimes parameters. Erroneous code example:
+
+```
+fn f() {}
```

Comment:
It works in the scenario when you're in a surrounding function or impl block which has defined lifetimes.